code/index.html

```diff
@@ -4,27 +4,103 @@
     <meta charset="utf-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <title>Project 3</title>
+    <link rel="stylesheet" href="./style.css">
     <!-- dont forget to add a css file and link it here! -->
   </head>
   <body>
     <!-- video or image as a header is cool :) -->
+   
+<header>
+  <div class="top-section-container">
+    <img class="top-section-logo" src="./Assets/Logo/Perfect gym.png" width="60px" height="60px">
```

Comment:
Both files and directories should be named with lowercase letters, so rather :

```
src="./assets/logo/perfect gym.png"
```

Moreover, styles should be applied only via class in CSS file. Let's fix that